push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/token/TokenLoader.java

```diff
@@ -107,7 +98,6 @@ public void loadAndQueueTokenBatch(@Observes @Dequeue MessageHolderWithVariants
         final Collection<Variant> variants = msg.getVariants();
```

Comment:
Ah... the damn `@link` did fool me - otherwise, with unused import, we have checkstyle fails ;-) 

will change - good catch, thanks!